src/payment/cart/OrderSummary.jsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { sendTrackEvent } from '@edx/frontend-analytics';
 import { FormattedMessage } from '@edx/frontend-i18n';
 
 import { utils } from '../../common';
 
 class OrderSummary extends React.Component {
   componentDidMount() {
     utils.markPerformanceIfAble('Order Summary component rendered');
+    sendTrackEvent('edx.bi.ecommerce.payment_mfe.order_summary_rendered');
```

Comment:
The timestamps on these events are pretty coarse, could we include the milliseconds from the start of the waterfall or something in the event?

Reply:
I'll add `performance.now()`. 

`The current high resolution time is the high resolution time from the time origin to the present time (typically called "now").`

I was mainly interested in the included timestamps for getting timing between the backend and frontend. Do you think they have high enough resolution for that?

Reply:
I suspect not. Segment makes an HTTP request to their infrastructure when you call `analytics.track()`. The server records the timestamp of the event during the processing of that request. The clock from the client is not trusted.

For this reason, I suspect that two events fired very near one another (milliseconds apart) could actually be processed out-of-order.

Reply:
Hmph, ok thanks for the info. I suppose I can just do `Date.now() / 1000` and `time.time()` for the mfe and backend respectively, but that just feels _dirty_